test/node/features/koa.feature

```diff
@@ -58,27 +64,32 @@ Scenario: an error thrown before the requestHandler middleware
   And the exception "message" equals "nope"
   And the exception "type" equals "nodejs"
   And the "file" of stack frame 0 equals "scenarios/app.js"
+  And the event "metaData.error_handler.before" is true
+  And the event "metaData.error_handler.after" is null
 
 Scenario: throwing non-Error error
-  Then I open the URL "http://koa/throw-non-error"
+  Then I open the URL "http://koa/throw-non-error" and get a 500 response
   And I wait to receive an error
   Then the error is valid for the error reporting API version "4" for the "Bugsnag Node" notifier
   And the event "unhandled" is true
   And the event "severity" equals "error"
   And the event "severityReason.type" equals "unhandledErrorMiddleware"
-  And the exception "errorClass" equals "InvalidError"
-  And the exception "message" matches "^koa middleware received a non-error\."
+  And the exception "errorClass" equals "Error"
+  And the exception "message" equals 'non-error thrown: \\"error\\"'
```

Comment:
As we rely on Koa catching errors now, it will internally wrap them in an Error instance before they reach Bugsnag (see [Koa's `context.js`](https://github.com/koajs/koa/blob/698ce0afbfac6480400625729a4b8fc4b4203fdc/lib/context.js#L119)). This changes the error class and message, but is otherwise equivalent